axlearn/common/host_array_test.py

```diff
@@ -12,6 +12,7 @@
 from absl.testing import absltest, parameterized
 from jax import numpy as jnp
 from jax.experimental import mesh_utils
+import pytest
```

Comment:
is this necessary?

Reply:
https://github.com/ptoulme-aws/axlearn/blob/52d0991ed0224f034ccf82b088d7e39b112cd302/axlearn/common/host_array_test.py#L68